nextflow.config

```diff
@@ -1,8 +1,8 @@
 manifest {
-    name = 'cool name nf'
-    description = 'nice description'
+    name = 'savio alignments nextflow'
```

Comment:
Not really needed to mention `savio alignments nextflow` is needed, simply `savio-alignments-nf` should be enough :)

Reply:
okok I'll keep it simple